mixer/pkg/server/server.go

```diff
@@ -253,3 +253,13 @@ func (s *Server) Close() error {
 func (s *Server) Addr() net.Addr {
 	return s.listener.Addr()
 }
+
+// InventoryMap converts adapter inventory to a builder map.
```

Comment:
Is there a better place for this? This function doesn't fit the overall functionality that is offered by the server package.